marine.h

```diff
@@ -0,0 +1,21 @@
+//
```

Comment:
Remove the comment

Reply:
lol clion create's it by default i didn't even notice it